pyramid_swagger/tween.py

```diff
@@ -42,18 +42,21 @@ class Settings(namedtuple(
     [
```

Comment:
Going to definitely need some updated docs+changelog on this one. Can wait until we've finalized the implementation.